### DISCUSSION TITLE
Savitar is open source!

### DISCUSSION THREAD

**Discussion by jkoutavas** [jkoutavas/Savitar2#32] [Announcements]
An exciting new chapter for Savitar. Making the app open source gives the ability to support ongoing development.

**Comment by jkoutavas** [comment]
Chronological [Software Design Notes](https://github.com/jkoutavas/Savitar2/blob/master/docs/Savitar2DevNotes.md) are available for viewing.

**Comment by jmacdotorg** [comment]
Hi Jay,

It's good to run across Savatar again!

I have never built a Swift application before. Would it be feasible to add instructions on compiling and running the application from source to the README file?

**Reply by jkoutavas** [reply]
Hi, @jmacdotorg .

Nice to hear from you. Yes, I can provide such notes. Do you intend on modifying the app? A binary download is already available if you just want to test/use the latest version.

**Reply by jmacdotorg** [reply]
A binary download would suit my needs splendidly, in fact. I couldn’t find a link for it, though?

**Reply by jkoutavas** [reply]
@jmacdotorg Cool. Email me an email address that you would like to use to join the [AppCenter](https://appcenter.ms) to get an alpha build distribution. My email address is: jay at heynow dot com.
